modules/bibdk_provider/BibdkUser.php

```diff
@@ -568,10 +585,12 @@ public function deleteFavourite($username, $agencyid) {
   }
 
   public function saveFavouriteData($name, $agencyid, $data) {
+    // Force all data saved to be encrypted.
     $params = array(
       'oui:userId' => $name,
       'oui:agencyId' => $agencyid,
-      'oui:favouriteData' => $data
+      'oui:favouriteData' => bibdk_provider_encrypt_data($data),
+      'oui:encrypted' => 1,
```

Comment:
Igen med 1 i stedet for TRUE. Hvis der er en dybere mening så bare ignorer disse konstateringer.